src/content/faq/11_crashing.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Discord always crashes when opening DevTools
+tags: crashing, crashes, oom
+---
+
+Discord on Windows uses 32bit electron which has a hard memory limit of 2gb. Discord's memory management isn't great so it will already get up to like 1.5gb ram usage if you are in a lot of servers and have it open for a while
```

Comment:
> Discord's memory management isn't great so it will already get up to like 1.5gb ram usage

This reads horribly. Imo would read better like this:
"Discords memory management is inefficient, so it will easily reach upwards of 1GB RAM usage"